cli/commands/deploy.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const dotenv = require('dotenv');
+dotenv.load({ path: '.env.session' });
+dotenv.load({ path: '.env.yadda'});
```

Comment:
What happens if either of these files don't exist?  Does `dotenv.load` silently ignore them and move on, or does it throw an error?

Reply:
It silently ignores them and continues on its way.